src/my_functions_product1.py

```diff
@@ -0,0 +1,721 @@
+   print('Please enter two different labels in the target field: ')
+
+   input_str = 'Enter label 0:'
+   warn_str = 'Please enter a valid label within target field'
+   result_str = "Your target label 0 is "
+   target_label0 = input_from_user(input_str, warn_str, result_str, possible_set)
+
+   possible_set.remove(target_label0)
+
+   input_str = 'Enter label 1:'
+   warn_str = 'Please enter a valid lable within target field'
+   result_str = "Your target label 1 is "
+   target_label1 = input_from_user(input_str, warn_str, result_str, possible_set)
+   
+   ### Inputting user preference for Sensitive Attribute
+   
+   remaining_columns = all_columns.copy()
+   remaining_columns.remove(target_field)
+
+   if cprediction_field_exists == 'Y':
+       remaining_columns.remove(cprediction_field) 
+
+   possible_set = remaining_columns
+   print("Your possible Sensitive Attribute columns '{}'".format(possible_set))
+   input_str = 'Which column is your sensitive column?:'
+   warn_str = 'Please enter a valid column'
+   result_str = "Your sensitive attribute column is "
+   sensitive_field = input_from_user(input_str, warn_str, result_str, possible_set)
+    
+    ### Inputting user preference for Sensitive 0-1 classes
+   possible_set = set(your_data_df[sensitive_field].unique())
+
+
+   print('unique values from the sensitive class:{}'.format(possible_set))
+   print('Please enter two different classes in the sensisitve field: ')
+
+   input_str = 'Enter class 0:'
+   warn_str = 'Please enter a valid class within sensitive field'
+   result_str = "Your sensitive attribute class 0 is "
+   sensitive_class0 = input_from_user(input_str, warn_str, result_str, possible_set)
+
+   possible_set.remove(sensitive_class0)
+
+   input_str = 'Enter class 1:'
+   warn_str = 'Please enter a valid class within sensitive field'
+   result_str = "Your sensitive attribute class 1 is "
+   sensitive_class1 = input_from_user(input_str, warn_str, result_str, possible_set)
+    
+    ### Do you want to omit any column from your data?
+   possible_set = ['Y','N']
+   input_str = 'Do you want to omit any column from your data? Please enter Y or N:'
+   warn_str = 'Please enter Y or N'
+   result_str = "Do you have any column for your current prediction? You entered"
+   fields_to_delete = input_from_user(input_str, warn_str, result_str, possible_set)
+    
+   remaining_columns = all_columns.copy()
+   remaining_columns.remove(target_field)
+
+   if cprediction_field_exists == 'Y':
+       remaining_columns.remove(cprediction_field) 
+
+   columns_can_omitted = remaining_columns
+    
+   not_to_consider_fields = deleting_columns(fields_to_delete,columns_can_omitted)
+
+   return cprediction_field_exists,cprediction_field,target_field, sensitive_field, your_data_df, not_to_consider_fields,sensitive_class0,sensitive_class1,target_label0,target_label1
+#==============================================================================
+   
```

Comment:
There are a few functions for handling user input! Wondering if they could be provided as a json or yml instead of being interactive!
target_col:
cur_prediction_col:
cols_to_delete:
sensitive_features_col:

and a bunch of asserts for sanity checks